spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 }
 ```
 
-## Examples
```

Comment:
Why lose all the examples here? We might want to move but having them is quite useful - no?